modules/terraform/Makefile

```diff
@@ -40,6 +40,20 @@ else
 	[ -z "$$FAIL" ] || (echo "Terraform configuration needs linting. Run '$(TERRAFORM) fmt'"; echo $$FAIL; exit 1)
 endif
 
+terraform/fmt:
+ifeq ($(wildcard *.tf),)
+	@echo "* $@: No terraform files detected"
+else
+	@{ command -v terraform-0.13 && command -v terraform-0.14; } >/dev/null || { echo "* $@ requires binaries named terraform-0.13 and terraform-0.14" && false; }
+	@ # vert exits non-zero if any of the versions are not acceptable, so `|| [[ -n "$VERSION" ]]` for a real error check
+	@TF12=0.12.29; \
+	TF13=$$(terraform-0.13 version --json | jq -r .terraform_version); \
+	TF14=$$(terraform-0.14 version --json | jq -r .terraform_version); \
+	VERSION=$$(vert -s "$$(terraform-config-inspect --json . | jq -r '.required_core[]')" "$$TF12" "$$TF13" "$$TF14" | head -1) || [[ -n "$$VERSION" ]]; \
+	VERSION=$${VERSION:0:4}; \
+	terraform-$${VERSION} fmt -recursive .
```

Comment:
We have to be backward compatible on this one. 99% of terraform users use our binary name format, nor will they have or can we require they have vert. Build harness is not used exclusively by cloud Posse CI.

Also, to support new versions of terraform, we have very few places we must update, and most of those places do not base the logic on the major version. I am not eager to start down this path.

Reply:
This is a new target, so we do not need to be backwards compatible. It is meant to be run from within the Build Harness Docker image, and fails with a nice error message if the binaries are not present. 

We can talk about packaging this up into something DRY, but the bulk of it is copy/pasted from the `test-command` workflow.